lib/composer.json

```diff
@@ -13,7 +17,8 @@
     "require": {
         "php": ">=7.0.0",
         "phpgt/cssxpath": "dev-master#4fbe420aba3d9e729940107ded4236a835a1a132",
-        "phpmailer/phpmailer": "6.6.0"
+        "phpmailer/phpmailer": "6.6.0",
+        "simplepie/simplepie": "dev-fix-base-url"
```

Comment:
I've used the newest brach for test purposes. This needs to be change to `dev-master` or git tags.

Reply:
I would propose something like a freshrss-1.6.0 branch (and then freshrss-1.6.1 etc. in the future), where our few modifications are rebased or cherry-picked on top of upstream if necessary. Or something similar with tags if that can work in an acceptable manner.

Something like `dev-master` doesn't sound good at all if you want to check out FreshRSS 1.20.0 a few years down the line. ^_^